.github/workflows/tests.yml

```diff
@@ -0,0 +1,63 @@
+name: tests
+
+on:
+  # Runs every sunday
+  schedule:
+      - cron: '0 0 * * SUN'
+
+  # Runs for pull requests
+  pull_request:
+    branches:
+      - master
```

Comment:
We would want to require manual approval for the workflow when there is a pull request. Do we need extra configuration to make sure of that? Or is that configuration on the side of the self-hosted runner?